docs/repository_management.md

```diff
@@ -7,12 +7,14 @@ external contributors, but they are documented here for clarity and
 transparency. If any of these things are particularly troublesome or painful for
 your workflow, please reach out to us so we can prioritize a fix.
 
-NOTE: We are currently in the process of migrating our repository to be
-GitHub-first and hide the merging complexity in a separate `google` feature
-branch so that standard development workflows don't have to bear the cost for
-every contribution. During this part of the migration period, please direct PRs
-to the `google` branch (which will be marked as the default branch). See
-https://groups.google.com/d/msg/iree-discuss/F07vsG9Ah4o/uAIusKO-BQAJ.
+## Branches
+
+The default branch is called `main`. PRs should be sent there. We also have a
+`google` branch that is synced from the Google internal source repository. This
+branch is merged from and to `main` frequently to usptream any changes coming
```

Comment:
```suggestion
branch is merged from and to `main` frequently to upstream any changes coming
```
typo

Reply:
Done. Can't seem to get spellright working.